src/components/AlertMessage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import FlashMessage from 'react-flash-message'
+import { Message, Header } from 'semantic-ui-react'
+
+const AlertMessage = (props) => {
+  return (
+    <FlashMessage duration={4000} >
```

Comment:
Do you have a test for the 4 seconds delay?

Reply:
Good point, I'll add one.